platform/ios/app/LimeGreenStyleLayer.h

```diff
@@ -0,0 +1,5 @@
+#import <Mapbox/Mapbox.h>
+
+@interface LimeGreenStyleLayer : MGLOpenGLStyleLayer
```

Comment:
As far as I know, LimeGreenStyleLayer.h and LimeGreenStyleLayer.m would be the first files that are identical between iosapp and macosapp. Do you think we should keep two copies around or share the code in platform/darwin/app/?